concepts/type-assertion/about.md

```diff
@@ -1,3 +1,34 @@
-# About
+# About Type Assertions
```

Comment:
You wanted to also include the type switch in this concept if I understood correctly.